doc/secciones/04_planificacion.tex

```diff
@@ -1,8 +1,206 @@
 \chapter{Planificación}
 
-\section{Metodología utilizada}
+Como ya se ha dicho, el proyecto se ceñirá a las prácticas y principios de la metodología ágil, que se basa en 17 principios esbozados en \href{https://agilemanifesto.org/iso/es/manifesto.html}{el manifiesto ágil}. Estos principios están diseñados para garantizar la satisfacción del cliente mediante la entrega temprana y continua de valor, con un fuerte enfoque en la excelencia técnica, el buen diseño, la planificación y la simplicidad.
```

Comment:
Ágil no es una metodología https://agilemania.com/is-agile-a-framework-or-methodology

Reply:
Realizado en 8dfbbd8.